imagebuildah/chroot_symlink.go

```diff
@@ -50,7 +50,7 @@ func resolveChrootedSymlinks() {
 	os.Exit(status)
 }
 
-func resolveSymLink(rootdir, filename string) (string, error) {
+func ResolveSymLink(rootdir, filename string) (string, error) {
```

Comment:
I thought golang wanted a comment line above a public function like this, but it doesn't seem to be barking about it.

Reply:
It does, the linter will fail.